From e739696452772a872f13bddb3f356082e0de882c Mon Sep 17 00:00:00 2001
From: Vilas R K <vilas.r.k@intel.com>
Date: Fri, 26 Jul 2024 10:44:50 +0530
Subject: [PATCH] Add APEX version of thermal AIDL in product packages

Added APEX version of AIDL to product packages to enhance modularity
and update mechanisms.

Removed support for android.hardware.thermal AIDL IThermal (@1)
as it is no longer compatible with the requirements for A15.

Upgraded to android.hardware.thermal AIDL IThermal (@2)
to ensure compliance with A15 specifications.

Tracked-On: OAM-122217
Signed-off-by: Vilas R K <vilas.r.k@intel.com>
---
 groups/thermal/thermal-daemon/product.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/groups/thermal/thermal-daemon/product.mk b/groups/thermal/thermal-daemon/product.mk
index d0dddf9..956383d 100644
--- a/groups/thermal/thermal-daemon/product.mk
+++ b/groups/thermal/thermal-daemon/product.mk
@@ -5,4 +5,4 @@ PRODUCT_COPY_FILES += \
 	$(INTEL_PATH_COMMON)/thermal/thermal-daemon/thermal-cpu-cdev-order.xml:/vendor/etc/thermal-daemon/thermal-cpu-cdev-order.xml
 
 # Thermal Hal
-PRODUCT_PACKAGES += android.hardware.thermal@aidl-service.intel
+PRODUCT_PACKAGES += com.android.hardware.thermal.intel
-- 
2.45.2

